shared/components/PageHeader/PageHeader.js

```diff
@@ -46,7 +46,7 @@ export const PageHeader = ({
     <Panel.Header>
       <section className="header-wrapper">
         {breadcrumbItems.length ? (
-          <section className="fd-has-margin-bottom-s">
```

Comment:
why did we remove this class?

Reply:
I decided it looks better by making the high header a bit smaller. Also, it suggests that (which is true) the "path" to the resource is
```
<resource_type> /
<resource_name>
``` 
Previously it was more like
```
<resource_type> /


<resource_name>
```



I can change that back if you want 😃